examples/react-18/src/App.js

```diff
@@ -0,0 +1,123 @@
+import PayloadReact from 'payload-react'
+import {
+  CardCode,
+  CardNumber,
+  Expiry,
+  PayloadInput,
+  PaymentForm,
+} from 'payload-react'
+import React, { useState } from 'react'
+
+import './App.css'
+
+function Example1() {
+  return (
+    <div>
+      {/* eslint-disable-next-line */}
+      <PayloadReact.form.payment
+        clientToken={process.env.REACT_APP_PAYLOAD_CLIENT_TOKEN}
+        className="container">
+        <PayloadReact.input.amount type="hidden" value="10.00" />
+        <div className="row pt-2">
+          <div className="form-group col7 px-1">
+            <label>Card</label>
+            {/* eslint-disable-next-line */}
+            <PayloadReact.input.card className="form-control" />
+          </div>
+        </div>
+        <div className="row pt-2">
+          <button className="btn btn-primary" type="submit">
+            Pay Now
+          </button>
+        </div>
+      </PayloadReact.form.payment>
```

Comment:
Why do we still need these lowercase references?

Reply:
This one I wrote it before @ianhalpern brought the new components.